dozer-log/src/errors.rs

```diff
@@ -23,4 +23,6 @@ pub enum ReaderError {
     Storage(#[from] crate::storage::Error),
     #[error("Reader thread has quit: {0:?}")]
     ReaderThreadQuit(#[source] Option<tokio::task::JoinError>),
+    #[error(transparent)]
+    ReaderBuilderError(#[from] ReaderBuilderError),
```

Comment:
Remove this. I intentionally differentiate between the building phase error and running phase error.